roles/docker/defaults/main.yml

```diff
@@ -15,12 +15,16 @@ docker_dns_servers_strict: yes
 
 docker_container_storage_setup: false
 
-#CentOS/RedHat docker-ce repo
+# CentOS/RedHat docker-ce repo
 docker_rh_repo_base_url: 'https://download.docker.com/linux/centos/7/$basearch/stable'
 docker_rh_repo_gpgkey: 'https://download.docker.com/linux/centos/gpg'
-#Ubuntu docker-ce repo
+# Ubuntu docker-ce repo
 docker_ubuntu_repo_base_url: "https://download.docker.com/linux/ubuntu"
 docker_ubuntu_repo_gpgkey: 'https://download.docker.com/linux/ubuntu/gpg'
-#Debian docker-ce repo
+# Debian docker-ce repo
 docker_debian_repo_base_url: "https://download.docker.com/linux/debian"
 docker_debian_repo_gpgkey: 'https://download.docker.com/linux/debian/gpg'
+
+# Use ansible_distribution_release for checking
+docker_nvidia_suppored_distributions:
```

Comment:
typo?
docker_nvidia_suppored_distributions -> docker_nvidia_supported_distributions
Is this right?

Reply:
so this is docker_nvidia_suppor(t)ed_distributions is removed from current code